src/models/Links.php

```diff
@@ -162,11 +160,11 @@ public function readRelated(): array
     }
 
     /**
-     * Get links from an id
-     *
-     * @param int $id
-     * @return array
-     */
+      * Get links from an id
```

Comment:
indentation of this doc comment was good before!

Reply:
I screwed up in between. Should be reverted to the good indentation from before. Sorry.